spec/app_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../app'
```

Comment:
Recommend that you drop the double for the app class and test it purely as an integration, especially as your repository classes have fully unit tested the behaviour of the items and orders.